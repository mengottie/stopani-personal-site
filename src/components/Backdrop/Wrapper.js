import styled from 'styled-components';

import { modal } from '../../styles/tokens';

const Backdrop = styled.div`
    position: fixed;
    top: 0;
    left: 0;
    width: 100%;
    height: 100%;
    z-index: 100;
    background-color: ${modal};
`

export default Backdrop;
