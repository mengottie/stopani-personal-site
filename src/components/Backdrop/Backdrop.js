import React from 'react';

import Wrapper from './Wrapper';

function Backdrop({ toggleSideMenuHandler }) {
    return(
        <Wrapper onClick={toggleSideMenuHandler} />
    )
}

export default Backdrop;
