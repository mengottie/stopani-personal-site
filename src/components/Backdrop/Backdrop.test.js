import React from 'react';
import { create } from 'react-test-renderer';

import Backdrop from './Backdrop';

jest.mock('./Backdrop', () => 'Backdrop');

describe('Backdrop', () => {
    let component = create(<Backdrop />);

    it('renders correctly', () => {
        expect(component.toJSON()).toMatchSnapshot();
    })
})
