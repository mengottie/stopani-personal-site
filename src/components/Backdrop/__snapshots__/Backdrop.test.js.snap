// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Backdrop renders correctly 1`] = `<Backdrop />`;
